velox/dwio/dwrf/writer/FlushPolicy.cpp

```diff
@@ -16,29 +16,51 @@
 
 #include "velox/dwio/dwrf/writer/FlushPolicy.h"
 
+static constexpr size_t kNumDictioanryTestsPerStripe = 3UL;
```

Comment:
Put in anonymous namespace? And briefly comment this control parameter? Thanks!

Reply:
My understanding is that cpp files will not be included the same way header files are. So they should not pollute the namespace and only have file scope.

Reply:
We still put static member in an anonymous namespace in cpp.